packages/DataTable/src/helpers/convertSizeStringToNumber.ts

```diff
@@ -0,0 +1,72 @@
+const pxRegex = /^(?<numberInPx>\d+)px$/;
+const vwRegex = /^(?<numberInVw>\d+)vw$/;
+const vhRegex = /^(?<numberInVh>\d+)vh$/;
+const calcWithVwRegex = /^calc\((?<numberInVw>\d+)vw(?<operator>(\+|-|\*|\/))(?<numberInPx>\d+)px\)$/;
+const calcWithVhRegex = /^calc\((?<numberInVh>\d+)vh(?<operator>(\+|-|\*|\/))(?<numberInPx>\d+)px\)$/;
+
+export enum Direction {
+  width = "width",
+  height = "height",
+}
+
+const DirectionToGroupSelector = {
+  [Direction.width]: "numberInVw",
+  [Direction.height]: "numberInVh",
+};
+
+const RegexSets = {
+  [Direction.width]: [pxRegex, vwRegex, calcWithVwRegex],
+  [Direction.height]: [pxRegex, vhRegex, calcWithVhRegex],
+};
+
+function calculateViewportSizeToPx(viewportSize: number, direction: Direction) {
+  return (window.visualViewport[direction] / 100) * viewportSize;
+}
+
+function calculateCalcToPx(left: number, right: number, operator: string) {
+  switch (operator) {
+    case "+":
+      return left + right;
+    case "-":
+      return left - right;
+    default:
+      return left;
+  }
+}
+
+export default function convertSizeStringToNumber(sizeStr: string, direction: Direction): number {
+  const regexSet = RegexSets[direction];
+  let result = 0;
+
+  regexSet.find((regex, index) => {
+    const match = regex.exec(sizeStr);
+
+    if (!match || !match.groups) return false;
+
+    switch (index) {
+      case 0:
+        result = Number.parseInt(match.groups.numberInPx, 10);
+        break;
+      case 1:
+        result = calculateViewportSizeToPx(
+          Number.parseInt(match.groups[DirectionToGroupSelector[direction]], 10),
```

Comment:
So then `convertSizeStringToNumber("100", "width")` should still work and return `100`, right?  

Is that a use case we want to _explicitly_ support, and indicate in the docs?
https://github.com/acl-services/paprika/pull/1228/files#diff-0c720a400684e34f0781322411e80251bc906ed4766234c4ca6b4a947a123e96R52-R56

Reply:
`"100"` will not work, the string has to match one of the regexes, but we can make it work for sure, we can create a follow-up ticket to support more format e.g. "100" "100%" ...